main.go

```diff
@@ -36,19 +37,28 @@ var (
 // GlobalFlags has command line flags to configure the exporter.
 type GlobalFlags struct {
 	CollStatsCollections  string `name:"mongodb.collstats-colls" help:"List of comma separared databases.collections to get $collStats" placeholder:"db1.col1,db2.col2"`
-	DSN                   string `name:"mongodb.dsn" help:"MongoDB connection URI" placeholder:"mongodb://user:pass@127.0.0.1:27017/admin?ssl=true"`
-	ExposePath            string `name:"expose-path" help:"Metrics expose path" default:"/metrics"`
+	URI                   string `name:"mongodb.uri" help:"MongoDB connection URI" placeholder:"mongodb://user:pass@127.0.0.1:27017/admin?ssl=true"`
+	WebTelemetryPath      string `name:"web.telemetry-path" help:"Metrics expose path" default:"/metrics"`
 	IndexStatsCollections string `name:"mongodb.indexstats-colls" help:"List of comma separared databases.collections to get $indexStats" placeholder:"db1.col1,db2.col2"`
 	ExposePort            int    `name:"expose-port" help:"HTTP expose server port" default:"9216"`
-	CompatibleMode        bool   `name:"compatible-mode" help:"Enable old mongodb-exporter compatible metrics"`
-	Debug                 bool   `name:"debug" short:"D" help:"Enable debug mode"`
+	CompatibleMode        bool   `name:"compatible-mode" help:"Enable old mongodb-exporter compatible metrics" default:"true"`
 	Version               bool   `name:"version" help:"Show version and exit"`
+
+	// To make this exporter a drop-in replacement of origimal one
+	LogLevel             string `name:"log.level" help:"Only log messages with the given severuty or above. Valid levels: [debbug, info, warn, error, fatal]" enum:"debbug,info,warn,error,fatal" default:"error"`
+	Test                 bool   `name:"test" help:"Check MongoDB connection, print BuildInfo() information and exit. (Not implemented yet)"`
+	CollectCollection    bool   `name:"collect.collection" help:"Enable collection of Collection metrics. (Deprecated. Use --mongodb.collstats-colls)"`
```

Comment:
let's remove flags that we don't use.

Reply:
That we don't use in PMM 2.9. We want new exporter flags be compatible with what PMM currently uses, but not port _all_ old flags

Reply:
I am going to remove the `test` flag but we are using the others in pmm-managed
```
                "--collect.collection",
		"--collect.database",
		"--collect.topmetrics",
		"--no-collect.connpoolstats",
		"--no-collect.indexusage",
		"--web.listen-address=:" + tdp.left + " .listen_port " + tdp.right,
```
https://github.com/percona/pmm-managed/pull/441/files#diff-c7dd80a44550716ef718d9b0eb3c38faR70-R76
We are not using them the same way, but pmm-managed use these flags.

Reply:
Then we have to have _some_ version-selection logic in pmm-managed

Reply:
No. If we don't implement https://jira.percona.com/browse/PMM-6450 this agent is parameters-compatible with the old one

Reply:
But you've already implemented it and we already merged new mongodb_exporter build logic. So we can remove unnecessary flags and just update flags list.